package.json

```
@@ -14,6 +14,7 @@
   },
   "scripts": {
     "serve": "cross-env VITE_IS_ELECTRON=false vite",
+    "serve:browser": "cross-env VITE_IS_ELECTRON=false VITE_IS_BROWSER=true vite",
```

```suggestion
    "browser:serve": "cross-env VITE_IS_ELECTRON=false VITE_IS_BROWSER=true vite",
```
electron:serveのように、browserを先に持ってきた方が統一感が出る気がしました。

また、スマホ版（Voicevox Mobile）ではVITE_IS_ELECTRONをVITE_TARGET=electronにしてます。こっちの方が読みやすくなるかも？